gatsby-browser.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/browser-apis/
+ */
+
+// You can delete this file if you're not using it
```

Comment:
Is this something that's likely going to be used in the near future? Might be worth deleting just to keep the root directory cleaner where possible.

Reply:
We can delete these files!
Was just an oversight on my part; will fix in PR resubmission

Reply:
@MJJBennett resolved with [this commit](https://github.com/ufvcsa/ufvcsa.github.io/pull/16/commits/203538091a46b8ddce896dfc1dfbb03087bfbea9)